doctr/models/predictor/pytorch.py

```diff
@@ -70,6 +84,11 @@ def forward(
 
         boxes, text_preds = self._process_predictions(loc_preds, word_preds)
 
+        # Rotate back pages and boxes while keeping original image size
+        if self.straighten_pages:
+            boxes = [rotate_boxes(page_boxes, angle, orig_shape=page.shape[:2], mask_shape=mask) for
```

Comment:
same here, the `.shape` indexing won't work properly on pytorch tensor which are channel first